modules/apps/27-interchain-accounts/host/types/params.go

```diff
@@ -57,3 +65,12 @@ func validateEnabled(i interface{}) error {
 
 	return nil
 }
+
+func validateAllowlist(i interface{}) error {
```

Comment:
Is it possible to have a []string slice with a empty entry?

Such as: []string{bankURL, "", stakingURL}?

Reply:
Good point, added a check